### DISCUSSION TITLE
Source file format in WIKI Single File Updater

### DISCUSSION THREAD

**Discussion by tobcro** [rfennell/AzurePipelines#2019] [Q&A] [answered]
Hi,
I am using WIKI Single File Updater to automate some tasks in release pipelines. I have one idea where I would like to copy a file from the wiki to a new file in the same wiki. When I try to do that with the "Data is a file" and "Source file" it appears that a real file path is expected (i also looked in the code). Does anyone know if there is any way around this or should I just find another way to solve it?

**Comment by rfennell** [comment, marked as answer]
The task was never designed to do, I suspect the assumptions made with relative paths will stop it working

I suspect it would be easier [run Git command within a custom script](https://learn.microsoft.com/en-us/azure/devops/pipelines/scripts/git-commands?view=azure-devops&tabs=yaml)

**Reply by tobcro** [reply]
Thank you for your clarification! I will try using a custom script.
